extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/SSLConfigHelper.java

```diff
@@ -0,0 +1,120 @@
+package io.quarkus.vertx.core.runtime;
+
+import io.quarkus.vertx.core.runtime.config.JksConfiguration;
+import io.quarkus.vertx.core.runtime.config.PemKeyCertConfiguration;
+import io.quarkus.vertx.core.runtime.config.PemTrustCertConfiguration;
+import io.quarkus.vertx.core.runtime.config.PfxConfiguration;
+import io.vertx.core.net.JksOptions;
+import io.vertx.core.net.KeyCertOptions;
+import io.vertx.core.net.PemKeyCertOptions;
+import io.vertx.core.net.PemTrustOptions;
+import io.vertx.core.net.PfxOptions;
+import io.vertx.core.net.TCPSSLOptions;
+
+public class SSLConfigHelper {
+
+    public static void configurePemTrustOptions(TCPSSLOptions options, PemTrustCertConfiguration configuration) {
```

Comment:
@michalszynkiewicz Don't we have the same type of code in gRPC? Maybe we can reuse the same classes.

Reply:
@cescoffier can we do any follow-up work related to gRPC in another PR? If you and @gsmet are both fine with this one, it would be great to merge it so I can rebase #9644

Reply:
Yes @tsegismont, this is not an issue with this PR, it's a good idea to reuse them!